Article/Article.js

```diff
@@ -26,4 +26,10 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+
+articles.forEach(article => {
+  const articleInstance = new Article(article);
```

Comment:
```suggestion
 new Article(article);
```
You don't need this assignment, it's not doing anything. If you try to console.log it outside the forEach it doesn't exist